core/src/main/scala/neotypes/DeferredQuery.scala

```diff
@@ -142,9 +281,19 @@ final class DeferredQueryBuilder private[neotypes] (private val parts: List[Defe
       )
   }
 
+  /** Add together the Query and Param of two DeferredQueryBuilders
```

Comment:
For this one and the other that receives a string, it would be better to say that it _concats_ together two queries. Instead of saying it adds.